etc/bento.env

```diff
@@ -203,7 +203,7 @@ BENTOV2_WES_WORKFLOW_TIMEOUT=172800
 
 # DRS
 BENTOV2_DRS_IMAGE=ghcr.io/bento-platform/bento_drs
-BENTOV2_DRS_VERSION=0.10.2
+BENTOV2_DRS_VERSION=0.12.2
```

Comment:
This one errors because of an Actions [error](https://github.com/bento-platform/bento_drs/actions/runs/5615547756) that prevented the publication of the image.
I'll re-run the job to see if this was a one off

Reply:
Yeah the job was started when Github Actions was down

Reply:
will bump to 0.12.3 when the patch for error-masking gets merged